docs/adding_tests.md

```diff
@@ -0,0 +1,82 @@
+# How to add tests
```

Comment:
I wanted to give a full explanation on how to introduce a new test with the new deployment method. I felt like this does not quite fit the topics covered in the other documentation files, so i decided on moving this explanation to a new file alltogether.